src/renderer/wg_engine/tvgWgGeometry.h

```diff
@@ -124,15 +124,15 @@ struct WgVertexBuffer {
     }
 
     // append source vertex buffer in index range from start to end (end not included)
-    void appendRange(const WgVertexBuffer& buff, size_t start_index, size_t end_index) {
-        if (start_index <= end_index)
-            for (size_t i = start_index; i < end_index; i++)
-                append(buff.vbuff[i]);
-        if (start_index > end_index) {
+    void appendRange(const WgVertexBuffer& buff, size_t start_index, size_t end_index, bool loop) {
```

Comment:
make a loop value false by default

Reply:
So please, at least mention it in comments

Reply:
@SergeyLebedkin please check https://github.com/thorvg/thorvg/pull/2771, should be more general